### ISSUE TITLE
implement Undistorted Focal Plane methods

### ISSUE THREAD

**Issue by AstroKEW78** [USGS-Astrogeology/SensorUtils#54]
Undistorted Focal Plane X, Y, Z - Camera::DistortionMap->UndistortedFocalPlaneX/Y/Z

**Comment by jessemapel** [comment]
I don't think we can do this. The CSM API does not allow access to the "inbetween" values like ISIS cameras do. When you do image to ground all you have is the image pixel and ground point, nothing in between (besides the look vector).

**Comment by jlaura** [comment]
What is ISIS using these values for?
